tools/tune/eq/gui.m

```diff
@@ -1,13 +1,104 @@
-clear;
+function gui()
+  pkg load signal;
+  clear;
+  EQ = [];
+  Fs = get_config().Fs;
```

Comment:
Good! I should have started reading this starting from the latest commits.

Reply:
Yea, I should try and squash as much of the old stuff out as possible. Shall this be resolved?

Reply:
Sure!